generator/src/main.rs

```diff
@@ -546,6 +547,11 @@ impl Parser {
     }
 
     fn parse_struct(&mut self, attrs: &[OwnedAttribute]) {
+        const STRUCTS_BLACKLIST: [&str; 3] = [
+            "XrCompositionLayerPassthroughHTC",
+            "XrPassthroughColorHTC",
+            "XrPassthroughHTC",
+        ];
```

Comment:
This might be the worst part.

Reply:
Is there a way we could rely on `disabled_exts` exclusively to mask these?

Reply:
I added `XR_HTC_passthrough` to `disabled_exts` but it wasn't sufficient. The xml structure is a bit backwards in this regard as the types are defined before the extension is named and the extension is only referring to these by name. We would need to parse the extensions before the structs and dynamically build a blacklist or remove the structs after parsing the extensions.

Reply:
Is it the parsing that fails, or the generator? If the latter, then removing structs after parsing extensions seems like a somewhat more maintainable solution.

Reply:
The generator completes but the build fails afterwards so we should be able to prune the list of structs after parsing is done.

Reply:
That makes sense, and sounds like an ideal balance. Users being able to fall back on the unsafe raw bindings is a key design goal.